client/express-checkout/index.js

```diff
@@ -526,48 +521,6 @@ jQuery( ( $ ) => {
 			wcpayECE.init();
 		},
 
-		blockExpressCheckoutButton: () => {
```

Comment:
Moved basically without changes to the `expressCheckoutButtonUi.blockButton()` method.